torch/csrc/jit/autodiff.cpp

```diff
@@ -149,6 +159,32 @@ static std::vector<Value*> gradientForNode(Node* node, ArrayRef<Value*> grad_val
           return returned_grads;
         }
       }
+      case aten::addmm: {
+        JIT_ASSERT(inputs.size() == 3);
+        JIT_ASSERT(outputs.size() == 1);
+        JIT_ASSERT(grads.size() == 1);
```

Comment:
I think you can assume that `outputs().size() == grads.size()` at this point. We dont want to check this in every single derivative formula